src/cdk/a11y/live-announcer/live-announcer.ts

```diff
@@ -25,7 +25,31 @@ import {
   LIVE_ANNOUNCER_ELEMENT_TOKEN,
   LIVE_ANNOUNCER_DEFAULT_OPTIONS,
 } from './live-announcer-tokens';
+import {TaskQueue} from './task-queue';
 
+/**
+ * The amount of time to wait between adding an announcer into the DOM and
+ * setting the text to it (causing it to be read). Notes on this value:
+ * This 100ms timeout is necessary for some browser + screen-reader combinations:
+ * - Both JAWS and NVDA over IE11 will not announce anything without a non-zero timeout.
+ * - With Chrome and IE11 with NVDA or JAWS, a repeated (identical) message won't be read a
+ *   second time without clearing and then using a non-zero delay.
+ * (using JAWS 17 at time of this writing).
+ */
+const ARIA_LIVE_ANNOUNCER_DELAY_MS = 100;
+
+/**
+ * The amount of time after a live announcement is added to the DOM before it's
+ * removed. This keeps the announcement node in the DOM for long enough for the
+ * screen reader to pick it up and announce it, but it doesn't linger on the
+ * page as the user continues browsing to the end of the document. From testing,
+ * queued live announce messages are read, even if the node containing them is
+ * removed before the message starts playing.
+ */
+const ARIA_LIVE_ANNOUNCER_REMOVAL_DELAY_MS = 1000;
```

Comment:
Will screen-readers continue reading a long message if it's removed from the DOM before the announcement is done? I worry that some will and that one second is too short.